src/resource/ensemble.rs

```diff
@@ -1,9 +1,17 @@
 //! An ensemble of multiple predictive models.
 
+use std::collections::HashMap;
+
 use super::Resource;
 use super::id::*;
 use super::status::*;
 
+/// List of field codes mapped to input fields
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct EnsembleField {
+  pub name: String,
+}
```

Comment:
In order to make this struct extensible without breaking semver, we should add a field of the form:

```
            /// Having one hidden field makes it possible to extend this struct
            /// without breaking semver API guarantees.
             #[serde(default, skip_serializing)]
            _hidden: (),
```